daemon-tests/src/lib.rs

```diff
@@ -462,6 +485,30 @@ impl Taker {
             .await
             .unwrap();
     }
+
+    /// Appends an event that overwrites the current DLC
+    ///
+    /// Note that the projection does not get updated, this change only manipulates the database!
+    /// When triggering another rollover this data will be loaded and used.
+    pub async fn simulate_previous_rollover(
```

Comment:
`simulate_one_rollover_ahead` might be a better name?

Reply:
Yeah, I'm not proud of the nifty trick, I just wanted to make sure to reproduce it and see the test turn green. 
In the long run I think it'd be better to test this by removing data rather than adding, but that's currently not easily possible.

Reply:
`append_rollover` sounds good too!

Reply:
I think it's great that you find a clever workaround. Otherwise we wouldn't have a test!